packages/odyssey-storybook/src/components/Infobox/Infobox.mdx

```diff
@@ -62,32 +59,44 @@ Avoid overusing Infobox within the same view. They are intended to draw the eye,
 
 It is ideal to direct users toward an appropriate action, especially for addressing errors. Infoboxes allow for both inline and separated actions. To preserve clarity, limit Infoboxes to one link.
 
-**STORY MISSING**
+<Canvas>
+  <Story id="components-infobox--with-link-inline" />
+</Canvas>
 
 The actions section is limited to links. If it's necessary to provide the user with an action, please direct them to the appropriate flow instead of beginning a new process inline.
 
-**STORY MISSING**
+<Canvas>
+  <Story id="components-infobox--with-link" />
+</Canvas>
 
 ### Placement
 
 Infoboxes should be displayed above the content they apply to but not higher than their scope.
 
 For example, a Form error should be displayed above all Fieldsets, but below the Form title.
 
-**STORY MISSING**
+<Canvas className="sb-do">
+  <Story id="components-infobox--form-do" />
+  <span class="sb-marker sb-marker--do">
+    <CheckIcon /> Do
+  </span>
+</Canvas>
```

Comment:
this looks great we'll just want a way to make it shareable! I imagine we could write a new component `DoOrDoNot` (better name plz) that accepts a story id prop `components-infobox--form-do` in this case, and a `do` or `doNot` prop the control the surrounding UI?

Reply:
That'd be great. Maybe we can extend from `Canvas` to allow for a `variant` prop or something? Then instead of applying a `className` we could say `variant="do"`.

I've created [OKTA-490546](https://oktainc.atlassian.net/browse/OKTA-490546) to track this.